### ISSUE TITLE
AT freezing at no nurseries until z

### ISSUE THREAD

**Issue by rgreen83** [genbtc/AutoTrimps#127]
For a few weeks now (i think since 4.7) AT freezes and does nothing (no attacking, no building, etc.) on the no nurseries until z level. I have no idea why it is related to this setting but i've tried changing the level for NNuZ and it always freezes on that level. If i set NNuZ to -1, AT will start freezing at random levels around the 50s-60s. I can always get AT working again by simply hitting the maps button in game and it seems this might be the first time AT has completed a map in the current run since this always drops the block and other early dropped unique maps, idk if that is pertinent or not.

**Comment by rgreen83** [comment]
Seems to only happen if specced out of home detector mastery, which I am a long way from affording since it provides no value, other than to now avoid this bug.

Affecting others here https://www.reddit.com/r/Trimps/comments/847tvy/autotrimp_stops_around_zone_5070_and_i_have_to/

**Comment by genbtc** [comment]
Fair point, I will try to look into why this combination of settings results in this.

**Comment by Calypsia** [comment]
+1 to this.  AT also freezes for me at the no nurseries until z level.  Only seems to happen when "Auto Abandon" setting is on and works fine if it is instead set to "wait to travel".

Edit: looks like if I put a number in "nurseries pre-spire" it forces a map early on in game and stops this problem from happening.
